src/AddTodoForm.js

```diff
@@ -1,14 +1,38 @@
-import React from "react";
-
-function AddTodoForm() {
-    return (
-      <form>
-        <label htmlFor="todoTitle">Title</label>
-        <input type="text" id="todoTitle" />
-        <button type="submit">Add</button>
-      </form>
-    );
-  }
-  
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+  // Create a new state variable named todoTitle with an update function named setTodoTitle
+  const [todoTitle, setTodoTitle] = useState("");
+
+  // Create a new function called handleAddTodo
+  const handleAddTodo = (event) => {
+    // Prevent the default behavior of the form submit
+    event.preventDefault();
+
+    // Invoke the onAddTodo callback prop and pass todoTitle as an argument
+    onAddTodo(todoTitle);
+
+    // Log the value of todoTitle in the console
+    console.log("Todo Title:", todoTitle);
+
+    // Reset the form so the text input value is cleared
+    setTodoTitle("");
+  };
```

Comment:
This all works (along with using `onChange={(e) => setTodoTitle(e.target.value` in the `input`), but to more closely follow the instructions and not use that unneeded state, you're going to want to set up your function like this

```suggestion
const handleAddTodo = (event) => {
    // Prevent the default behavior of the form submit
    event.preventDefault();

    // Store todoTitle in a variable
    const todoTitle = event.target.title.value
    
    // Invoke the onAddTodo callback prop and pass todoTitle as an argument (using props instead of destructuring)
   props.onAddTodo(todoTitle);

    // Log the value of todoTitle in the console
    console.log("Todo Title:", todoTitle);

    // Reset the form so the text input value is cleared
    event.target.rest();
  };
```

Reply:
all these changes made errors in the code

Reply:
and what is event.target.rest(). ?

Reply:
oops that last bit is a typo on my part and should be `event.target.reset()`, which you can read up on [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset)

Reply:
I realized that after I sent the reply